js/reveal.js

```diff
@@ -292,6 +292,8 @@ export default function( revealElement, options ) {
 			statusElement.style.position = 'absolute';
 			statusElement.style.height = '1px';
 			statusElement.style.width = '1px';
+			statusElement.style.top = '-1px';
+			statusElement.style.left = '-1px';
```

Comment:
The status element is used for accessibility—it calls out text to a screen reader. It serves no purpose in PDF exports though, so maybe we can add `.aria-status { display: none; }` to the pdf print sheet instead of changing it here?

Reply:
Yup, seems reasonable. I'll do that.